src/lib/utils/concepts.h

```diff
@@ -12,6 +12,7 @@
 #include <compare>
 #include <concepts>
 #include <type_traits>
+#include <ostream>
```

Comment:
Either this was a include left over from debugging, or there is something I'm missing about how these concepts work.

Reply:
TL;DR: include what you use

The include is needed for the `streamable<>` concept. This was not introduced here, but the compiler needed the header when including `<botan/concepts.h>` in some compilation unit that didn't slurp it in through other ways.

Its likely that this particular PR wouldn't have needed it. Nevertheless I left it in for completeness.

Reply:
OK I was only looking at the diff and missed the preexisting concept. Yeah that's fine.